oxd-server/src/main/java/org/gluu/oxd/server/Cli.java

```diff
@@ -62,8 +59,17 @@ public static void main(String[] args) {
             RpService rpService = injector.getInstance(RpService.class);
             rpService.load();
 
+            //check multiple options
+            if(Cli.checkForMultipleOptions(cmd)) {
+                System.out.println("Multiple parameters in command is not allowed.");
+                printHelpAndExit();
+                return;
+            }
             // list
             if (cmd.hasOption("l")) {
+                if(Cli.checkForArgsAfterListParameter(args)){
+                    System.out.println("Arguments after list parameter is not required, hence will be ignored.");
```

Comment:
Lets add `Warning` word for clarity. `Warning: Arguments after list parameter is not required, hence will be ignored.`

Reply:
Warning added after in message.